core/templates/dev/head/domain/summary/ExplorationSummaryBackendApiService.ts

```diff
@@ -50,6 +50,12 @@ oppia.factory('ExplorationSummaryBackendApiService', [
       }).then(function(response) {
         var summaries = angular.copy(response.data.summaries);
         if (successCallback) {
+          if (!summaries) {
```

Comment:
`if (summaries === null)` just to catch the narrow case of null (which is what you are checking for) and not also catch any cases of implicit falsy conversion.

Reply:
Done.